src/js/utils/localStorage.ts

```diff
@@ -14,7 +18,7 @@ export const verifyData = (nObj, oObj) => {
   return Object.entries(oObj).every(([key, charts]) => verifyCharts(nObj[key], charts));
 };
 
-export const saveValue = (key, value) => {
+export const saveValue = (key: string, value: any) => {
   try {
     const serialized = JSON.stringify(value);
     localStorage.setItem(key, serialized);
```

Comment:
i know this isnt part of the changes but definitely at least log if setting local storage fails